libs/sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -952,7 +952,14 @@ public Void visitDropTable(DropTable<?> node, Integer indent) {
             if (node.dropIfExists()) {
                 builder.append("IF EXISTS ");
             }
-            node.table().accept(this, indent);
+            var tables = node.tables().iterator();
+            while (tables.hasNext()) {
+                tables.next().accept(this, indent);
+                if (tables.hasNext()) {
+                    builder.append(',');
+                }
+            }
+            builder.append(' ');
```

Comment:
Does this leave a trailing whitespace? If so, maybe change to:

```suggestion
                    builder.append(', ');
                }
            }
```